democracy_club/templates/for_voters.html

```diff
@@ -53,7 +53,7 @@ <h3>Get the Data</h3>
 
```

Comment:
I'm not actually sure this 'for_voters' template should exist any longer. It's not linked from anywhere on the DC site so perhaps the work here is to delete the template and add a redirect to the projects home page? https://stage.democracyclub.org.uk/voters/